lang/en/docs/tutorials/python-ml/train-ml-model.md

```diff
@@ -146,12 +131,12 @@ trained model for additional predictions on new data.
 The second result visible is `Machine Learning - Parity Plot`, which contains the predicted versus actual values for the
 adsorption energies we trained the model on.
 
-![Results Tab](../../images/tutorials/ml_train_results_tab.png "Results Tab")
+![Results Tab Showcasing Parity Plot](../../images/tutorials/pythonML/ml-train-results-tab.png "Results Tab Showcasing Parity Plot")
 
 ## Animation
 
 This tutorial is demonstrated in the following animation:
 
 <div class="video-wrapper">
-<iframe class="gifffer" width="100%" height="100%" src="https://www.youtube.com/embed/Eiw6bYn_w74" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<iframe class="gifffer" width="100%" height="100%" src="https://www.youtube.com/embed/zUqjHH0JCng" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Let's change the slide that is used for the video thumbnail pls to be more descriptive

Reply:
+ The video should not have the JSON editor artifact

Reply:
See above comment - phrasing has been changed. New version also includes the train/test split.